pw_mgr/lib/crypto_utils.py

```diff
@@ -0,0 +1,63 @@
+from cryptography.fernet import Fernet
+
+
+def write_key():
+    """
+    if key.key exists, load key.key. otherwise, create key.key.
+    """
+    with open ("key.key", "wb") as key_file:
+        key = Fernet.generate_key()
+        key_file.write(key)
+
+"""check if key.key exists. if not, create key.key. if yes, open key.key."""
+def init_key():
+    try:    
+        f = open("key.key")
```

Comment:
look at how I wrote the first test to check if the file is there.

Reply:
you're talking about using os.path.exists as an if/else statement, not using sys.path.append("."), right?

Reply:
correct, I am talking about os.path.exists  and avoid using the exception